geppetto-ui/src/components/html-viewer/showcase/HTMLViewerShowcase.less

```diff
@@ -0,0 +1,62 @@
+a {
```

Comment:
These styles are poisonous: may affect the whole showcase!

Reply:
A solution can be to use a parent class or id in order to not affect anything else